vault/logical_system_test.go

```diff
@@ -4791,7 +4792,10 @@ func TestSystemBackend_LoggersByName(t *testing.T) {
 		t.Run(fmt.Sprintf("loggers-by-name-%s", tc.logger), func(t *testing.T) {
 			t.Parallel()
 
-			core, b, _ := testCoreSystemBackend(t)
+			core, _, _ := TestCoreUnsealedWithConfig(t, &CoreConfig{
+				Logger: logging.NewVaultLogger(hclog.Trace),
+			})
+			b := core.systemBackend
```

Comment:
This doesn't look related to activity log stuff. Is this correct?

Reply:
This is a part of the "Deflake" commit: 28dd651c771ee495902762589c41456659767302